### PR TITLE
Handle non set order concepts

### PR DESCRIPTION
**Pull request by ojwanganto**
## Requirements

- [ ] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) label. See existing PR titles for inspiration.
- [ ] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://zeroheight.com/23a080e38/p/880723-introduction).
- [ ] My work includes tests or is validated by existing tests.

## Summary
Enhance the result form to document results for order concepts which are not sets or have no set members

## Screenshots
1. Complete blood count order (has set members)- 
image
2. Complete blood count result form - 
image
3. Viral load order result form (Has no set members) - 
image

### CODE REVIEW

File: src/results/result-form.resource.ts [hunk 1]
```diff
@@ -305,15 +315,34 @@ export function useGetOrderConceptByUuid(uuid: string) {
     { data: ConceptResponse },
     Error
   >(apiUrl, openmrsFetch);
+  console.log("Mini concept data: ", data);
```
Comment (jabahum) [inline src/results/result-form.resource.ts]: can we remove unnnecessry logs

File: src/results/result-form.resource.ts [hunk 2]
```diff
@@ -305,15 +315,34 @@ export function useGetOrderConceptByUuid(uuid: string) {
...
   };
 }
 
+export function useGetOrderConceptDetailsByUuid(uuid: string) {
+  const apiUrl = `/ws/rest/v1/concept/${uuid}?v=full`;
+  const { data, error, isLoading, isValidating, mutate } = useSWR<
+    { data: ConceptResponse },
+    Error
+  >(apiUrl, openmrsFetch);
+  console.log("log url", data);
```
Comment (jabahum) [inline src/results/result-form.resource.ts]: this too

File: src/results/single-result-form.component.tsx [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+    // assign value to test
+    let groupMembers = [];
+    let obsValue = [];
+    const ob = {
+      concept: { uuid: order.concept.uuid },
+      status: "FINAL",
+      order: { uuid: order.uuid },
+      groupMembers: groupMembers,
+    };
+    // concept.forEach((item) => {
```
Comment (jabahum) [inline src/results/single-result-form.component.tsx]: can we clean this too if not neccessary

File: src/work-list/work-list.resource.ts [hunk 4]
```diff
@@ -128,6 +128,9 @@ export function useGetOrdersWorklist(
     Error
   >(apiUrl, openmrsFetch);
 
+  console.log(
```
Comment (jabahum) [inline src/work-list/work-list.resource.ts]: this too

File: src/work-list/work-list.component.tsx [hunk 5]
```diff
@@ -126,7 +127,8 @@ const WorkList: React.FC<WorklistProps> = ({ fulfillerStatus }) => {
         onClick={() => {
           launchOverlay(
             t("resultForm", "Result  Tests"),
-            <ResultForm patientUuid={patientUuid} order={order} />
+            // <ResultForm patientUuid={patientUuid} order={order} />
```
Comment (jabahum) [inline src/work-list/work-list.component.tsx]: and this

File: src/results/result-form.resource.ts (hunk 1 above)
Comment (ojwanganto) [reply src/results/result-form.resource.ts]: I had removed this. Kindly check again

File: src/results/result-form.resource.ts (hunk 2 above)
Comment (ojwanganto) [reply src/results/result-form.resource.ts]: I already removed this also.

**Review by jabahum — APPROVED** [review]
LGTM thanks @ojwanganto

**Pull request merged**